plugins/symbol-processor/src/main/kotlin/org/jetbrains/dataframe/ksp/toJsonElement.kt

```diff
@@ -0,0 +1,53 @@
+package org.jetbrains.dataframe.ksp
+
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.buildJsonObject
+import org.jetbrains.kotlinx.dataframe.schema.ColumnSchema
+import org.jetbrains.kotlinx.dataframe.schema.DataFrameSchema
+
+/**
+ * Serializes data schema into a human-readable JSON format.
+ * Input of compiler plugin for "imported data schema" feature
```

Comment:
please add a tiny sample :) similar to what Nikita did in serialization_format.md It helps to see that this builds